TransformProcessor/Logic/vtkSlicerTransformProcessorLogic.cxx

```diff
@@ -441,6 +445,25 @@ void vtkSlicerTransformProcessorLogic::ComputeFullTransform( vtkMRMLTransformPro
   outputTransformNode->SetAndObserveTransformToParent( fromToToLinearTransformNode );
 }
 
+//----------------------------------------------------------------------------
+void vtkSlicerTransformProcessorLogic::ComputeInverseTransform( vtkMRMLTransformProcessorNode* paramNode )
+{
+  bool verboseWarnings = true;
+  bool conditionsMetForProcessing = this->IsTransformProcessingPossible( paramNode, verboseWarnings );
+  if ( conditionsMetForProcessing == false )
+  {
+    return;
+  }
+
+  vtkMRMLLinearTransformNode* forwardTransformNode = paramNode->GetInputForwardTransformNode();
```

Comment:
Use GetTransformBetweenNodes(). It is faster (does not go through world transform, just through the first common parent), works for arbitrary transforms (not just linear), and world transforms (if either or both transforms are NULL).